src/main/java/frc/team2412/robot/commands/climb/ClimbTestCommand.java

```diff
@@ -6,13 +6,8 @@
 public class ClimbTestCommand extends SequentialCommandGroup {
 
     public ClimbTestCommand(ClimbSubsystem subsystem) {
-        addCommands(new ExtendAngledHookCommand(subsystem),
-                new RetractAngledHookCommand(subsystem),
-                new UnangleClimbHookCommand(subsystem),
-                new NeutralClimbHookCommand(subsystem),
-                new AngleClimbHookCommand(subsystem),
-                new ExtendFixedHookCommand(subsystem),
-                new RetractFixedHookCommand(subsystem));
+        addCommands(new ExtendArmCommand(subsystem),
+                new RetractArmCommand(subsystem));
```

Comment:
Should probably test `FullRetractArmCommand` as well, with waits in between to give the arm enough time to physically move up and down.